kinto/core/statsd.py

```diff
@@ -6,13 +6,18 @@
 except ImportError:  # pragma: no cover
     statsd_module = None
 
+from pyramid.exceptions import ConfigurationError
 from six.moves.urllib import parse as urlparse
 
 from kinto.core import utils
 
 
 class Client(object):
     def __init__(self, host, port, prefix):
+        if statsd_module is None:
```

Comment:
Yes, but this client is instantiated only if `statsd_url` is specified in the settings (cf. `initialization.py`)